### PR TITLE
Referee gui

### PR DESCRIPTION
**Pull request by Oranshich**

### CODE REVIEW

File: src/main/java/GUI/FootballMain.java [hunk 1]
```diff
@@ -272,12 +279,15 @@ public static void showDialog(UI lastUI, String msg, String receiveType,StringBu
             newWindow.setCloseOnOutsideClick(false);
             newWindow.setCloseOnEsc(false);
             newWindow.setVisible(true);
+            newWindow.setWidth("500px");
```
Comment (galrosenthal) [inline src/main/java/GUI/FootballMain.java:282]: Why did you set this to specified size? 
you can just add a label and the label will get the max width it needs

File: src/main/java/GUI/RoleRelatedViews/Referee/RefereeControls.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        watchGamesBtn.addClickListener(e -> {
+
+            UI lastUI = UI.getCurrent();
+            VaadinSession se = VaadinSession.getCurrent();
+            String username = getUsernameFromSession();
+
+                UI.setCurrent(lastUI);
+                VaadinSession.setCurrent(se);
+                MainController.DisplayScheduledGame(username);
+
```
Comment (galrosenthal) [inline src/main/java/GUI/RoleRelatedViews/Referee/RefereeControls.java:42]: where is the Thread start part? did you test this button?

Comment (Oranshich) [reply src/main/java/GUI/RoleRelatedViews/Referee/RefereeControls.java:42]: He doesn't need a thread for displaying games

File: src/main/java/GUI/FootballMain.java (hunk 1 above)
Comment (Oranshich) [reply src/main/java/GUI/FootballMain.java:282]: It's not working without it

File: src/main/java/GUI/FootballMain.java [hunk 3]
```diff
@@ -488,6 +509,49 @@ private static void createMultiStringInputs(VerticalLayout verticalLayout, int n
         }
     }
 
+    public static void showModal(Collection<String>... displayValues){
+        Dialog newWindow = new Dialog();
+        newWindow.setCloseOnOutsideClick(false);
+        newWindow.setCloseOnEsc(false);
+        newWindow.setVisible(true);
+        newWindow.setWidth("500px");
```
Comment (galrosenthal) [inline src/main/java/GUI/FootballMain.java:517]: why do you still have a static px size?

**Pull request merged**